docs/using.rst

```diff
@@ -139,10 +139,12 @@ To use ADB with Host-Based tools:
    CLI, you can find it listed as a "client binary" download in the official
    `Docker Repositories <https://github.com/docker/docker/releases>`_.
 
-   **Note:** If you encounter a Docker client and server version mismatch such as :
-   `$ docker ps`
-   `Error response from daemon: client is newer than server (client API version: 1.21, server API version: 1.20)`
-   You will need to download an earlier compatible version of Docker for your host machine.  Docker release versions and docker API versions are not the same. Typically, you will need to try the previous release (i.e. if you get this error message using a docker 1.9 CLI, try a docker 1.8 CLI).
+   **Note:** If you encounter a Docker client and server version mismatch such as:: 
+
+    $ docker ps 
+    Error response from daemon: client is newer than server (client API version: 1.21, server API version: 1.20)
+
+   You will need to download an earlier compatible version of Docker for your host machine. Docker release versions and docker API versions are not the same. Typically, you will need to try the previous release (i.e. if you get this error message using a docker 1.9 CLI, try a docker 1.8 CLI).
 
 
    If you are using Eclipse, you should follow these steps:
```

Comment:
In step 2 and 3 of this section, it would be better if we highlight navigation "Windows -> show Views -> Console" as `Windows -> show Views -> Console` or some other good way.